includes/wf_crm_webform_preprocess.inc

```diff
@@ -615,13 +616,16 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
       'id' => 'wf-crm-billing-total',
       'data-amount' => $total,
     );
-    return theme('table', array(
-      'sticky' => FALSE,
-      'caption' => $this->contribution_page['title'],
-      'header' => array(),
-      'rows' => $rows,
-      'attributes' => array('id' => "wf-crm-billing-items"),
-    ));
+
+    $build = [
+      '#type' => 'table',
+      '#sticky' => FALSE,
+      '#caption' => $this->contribution_page['title'],
+      '#header' => array(),
+      '#rows' => $rows,
+      '#attributes' => array('id' => "wf-crm-billing-items"),
+    ];
+    return \Drupal::getContainer()->get('renderer')->render($build);
```

Comment:
Ugh. Similar to the thing about converting translation markup to strings early, is it possible to avoid the early render here and just return the render array? And, again, if doing so would be a big project, it should have an issue to address this later.

Reply:
It's a big project because it adds this to the `#prefix` or something. Not ideal, but I don't want to tackle it in this PR.

Reply:
:) This was fixed.